javascript/ql/lib/semmle/javascript/frameworks/NodeJSLib.qll

```diff
@@ -408,11 +408,13 @@ module NodeJSLib {
 
   /**
    * Holds if the `i`th parameter of method `methodName` of the Node.js
-   * `fs` module might represent a file path.
+   * `fs` module or the `fs-extra` might represent a file path.
    *
-   * We determine this by looking for an externs declaration for
+   * For `fs`, we determine this by looking for an externs declaration for
    * `fs.methodName` where the `i`th parameter's name is `filename` or
    * `path` or a variation thereof.
+   *
+   * For `fs-extra`, we use the `fsExtraExtensionFileParam` predicate.
```

Comment:
QLDoc should summarize the semantics, not the implementation. We only have a few technical corners of our libraries that need to refer explicitly to other predicates in their QLDoc.

```suggestion
   * For `fs-extra`, we use a manually maintained list.
```